SETUP/GRAPH.md

```diff
@@ -0,0 +1,138 @@
+# Graphs
+
+This file contains a description of the way graphs are created in dproofreaders.
+
+In case of problems, please post to the [DP Site Code](http://www.pgdp.net/phpBB3/viewforum.php?f=32)
+forum at the [primary DP site](https://www.pgdp.net/c).
+
+## Overview
+Graphs are drawn on the browser using a set of functions from [svgGraphs.js](../scripts/svgGraphs.js) that interact with the [d3js](https://d3js.org/) library to render [Scalable Vector Graphics (svg)](https://en.wikipedia.org/wiki/Scalable_Vector_Graphics) elements. The 4 types of graphs that are supported are
```

Comment:
Somewhere, probably in the Overview, we should talk a bit about embedded vs iframe graphs, at least in the sense that there are PHP pages that only contain the graph that can be used as iframes within a page whereas most of the graphs are rendered within a larger page.

My thinking is that in the future a developer might come in and say "I want to make this graph accessible as if it were an image" and being able to easily see where we've done this prior via iframe would be useful.